staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -120,6 +120,9 @@ func roundTripTypes(t *testing.T, scheme *runtime.Scheme, codecFactory runtimese
 		internalKindToGoType := scheme.KnownTypes(internalVersion)
 
 		for kind := range internalKindToGoType {
+			if kind != "PriorityClassList" {
```

Comment:
Why is this needed?

Reply:
Sorry, It‘s just for test. I deleted it.